solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1542 @@
+
+    event BridgeGovernanceTransferStarted(
+        address newBridgeGovernance,
+        uint256 timestamp
+    );
+    event BridgeGovernanceTransferred(address newBridgeGovernance);
+
+    event DepositDustThresholdUpdateStarted(
+        uint64 newDepositDustThreshold,
+        uint256 timestamp
+    );
+    event DepositDustThresholdUpdated(uint64 depositDustThreshold);
+
+    event DepositTreasuryFeeDivisorUpdateStarted(
+        uint64 depositTreasuryFeeDivisor,
+        uint256 timestamp
+    );
+    event DepositTreasuryFeeDivisorUpdated(uint64 depositTreasuryFeeDivisor);
+
+    event DepositTxMaxFeeUpdateStarted(
+        uint64 newDepositTxMaxFee,
+        uint256 timestamp
+    );
+    event DepositTxMaxFeeUpdated(uint64 depositTxMaxFee);
+
+    event RedemptionDustThresholdUpdateStarted(
+        uint64 newRedemptionDustThreshold,
+        uint256 timestamp
+    );
+    event RedemptionDustThresholdUpdated(uint64 redemptionDustThreshold);
+
+    event RedemptionTreasuryFeeDivisorUpdateStarted(
+        uint64 newRedemptionTreasuryFeeDivisor,
+        uint256 timestamp
+    );
+    event RedemptionTreasuryFeeDivisorUpdated(
+        uint64 redemptionTreasuryFeeDivisor
+    );
+
+    event RedemptionTxMaxFeeUpdateStarted(
+        uint64 newRedemptionTxMaxFee,
+        uint256 timestamp
+    );
+    event RedemptionTxMaxFeeUpdated(uint64 redemptionTxMaxFee);
+
+    event RedemptionTimeoutUpdateStarted(
+        uint64 newRedemptionTimeout,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutUpdated(uint64 redemptionTimeout);
+
+    event RedemptionTimeoutSlashingAmountUpdateStarted(
+        uint96 newRedemptionTimeoutSlashingAmount,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutSlashingAmountUpdated(
+        uint96 redemptionTimeoutSlashingAmount
+    );
+
+    event RedemptionTimeoutNotifierRewardMultiplierUpdateStarted(
+        uint64 newRedemptionTimeoutNotifierRewardMultiplier,
+        uint256 timestamp
+    );
+    event RedemptionTimeoutNotifierRewardMultiplierUpdated(
+        uint64 redemptionTimeoutNotifierRewardMultiplier
+    );
+
+    event MovingFundsTxMaxTotalFeeUpdateStarted(
+        uint64 newMovingFundsTxMaxTotalFee,
+        uint256 timestamp
+    );
+    event MovingFundsTxMaxTotalFeeUpdated(uint64 movingFundsTxMaxTotalFee);
+
+    event MovingFundsDustThresholdUpdateStarted(
+        uint64 newMovingFundsDustThreshold,
+        uint256 timestamp
+    );
+    event MovingFundsDustThresholdUpdated(uint64 movingFundsDustThreshold);
+
+    event MovingFundsTimeoutResetDelayUpdateStarted(
+        uint32 newMovingFundsTimeoutResetDelay,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutResetDelayUpdated(
+        uint64 movingFundsTimeoutResetDelay
+    );
+
+    event MovingFundsTimeoutUpdateStarted(
+        uint64 newMovingFundsTimeout,
+        uint256 timestamp
+    );
+    event MovingFundsTimeoutUpdated(uint64 movingFundsTimeout);
```

Comment:
The type of `movingFundsTimeout` Is `uint32`:
```
uint32 movingFundsTimeout;
```
Let's change it in every occurence (functions, events).

Reply:
Done